jenkinsfiles/grant_access_to_app.groovy

```diff
@@ -4,34 +4,25 @@ node {
         checkout scm
     }
 
-    // See details on broken pip https://askubuntu.com/a/528625
-    stage ("Set up python3 environment") {
-        // sh "python3 -m venv venv"
-        sh "python3 -m venv --without-pip venv"
-        sh "curl https://bootstrap.pypa.io/get-pip.py | venv/bin/python3"
-    }
-
-    stage ("Install script dependencies") {
-        sh "venv/bin/pip3 install -r jenkinsfiles/auth0-users/requirements.txt"
-    }
-
     // Creates passwordless user in Auth0 and adds to app group
     // NOTE: Groups provided by Auth0 Authorization Extension
     stage ("Create Auth0 passwordless user and add to app group") {
       withCredentials([
-        [$class: 'StringBinding', credentialsId: 'AUTH0_DOMAIN', variable:
-'AUTH0_DOMAIN'],
-        [$class: 'StringBinding', credentialsId: 'AUTH0_CLIENT_ID', variable:
-'AUTH0_CLIENT_ID'],
-        [$class: 'StringBinding', credentialsId: 'AUTH0_CLIENT_SECRET', variable:
-'AUTH0_CLIENT_SECRET'],
-        [$class: 'StringBinding', credentialsId: 'AUTHZ_API', variable:
-'AUTHZ_API']
+        usernamePassword(
```

Comment:
So with this syntax the credentials from Jenkins go into the environment variables?